src/clj_kondo/impl/analyzer.clj

```diff
@@ -2707,11 +2707,11 @@
           (binding [*out* *err*]
             (print ".") (flush))))
       (when file-analyzed-fn
-        (swap! files-analyzed-done inc)
+        (swap! files inc)
```

Comment:
The files atom needs to be incremented regardless whether we're having a file-analyzed-fn

Reply:
good catch!